NorthstarDLL/logging/logging.cpp

```diff
@@ -261,12 +261,18 @@ void StartupLog()
 		spdlog::info("Operating System: {} (Wine)", sysname);
 		spdlog::info("Wine build: {}", wine_get_build_id());
 
+		// STEAM_COMPAT_TOOL_PATHS is a colon separated lists of all compat tool paths used
+		// The first one tends to be the Proton path itself
```

Comment:
Tends to be? Doesn't this mean that this PR only fixes the cases where it was broken, but breaks cases where it wasn't?

Could we just look for the `NorthstarProton` + `:`/`0x00` substring to find it more consistently?

Reply:
> Tends to be? Doesn't this mean that this PR only fixes the cases where it was broken, but breaks cases where it wasn't?

Entries to the list are appended to the end, so when Proton steam puts it as the first entry.
But this isn't set in stone and may change in the future.

> Could we just look for the NorthstarProton + :/0x00 substring to find it more consistently?

NorthstarProton is not a requirement, only a suggestion.
GE-Proton works just as well (arguably even better).

a colon is only added between multiple entries.

Reply:
> NorthstarProton is not a requirement, only a suggestion.
> GE-Proton works just as well (arguably even better).

Fair enough. I guess we can't really guarantee 100% accuracy here then